src/tr/uzaymanga/src/eu/kanade/tachiyomi/extension/tr/uzaymanga/UzayManga.kt

```diff
@@ -149,7 +216,11 @@ class UzayManga : ParsedHttpSource() {
         document.selectFirst("div.grid h2 + p") != null
 
     private fun String.toDate(): Long =
-        try { dateFormat.parse(this)!!.time } catch (_: Exception) { 0L }
+        try {
+            dateFormat.parse(this)!!.time
+        } catch (_: Exception) {
+            0L
+        }
```

Comment:
use `dateFormat.tryParse(str)` when creating chapter instead of this function

import keiyoushi.utils.tryParse